apps/fabric-website/src/components/IconGrid/IconGrid.module.scss

```diff
@@ -15,12 +15,11 @@
     justify-content: space-around;
     padding: 8px;
     width: 80px;
-    overflow: hidden;
 
     span {
       color: $ms-color-neutralPrimary;
-      font-size: $ms-font-size-xs;
-      opacity: 0;
+      font-size: $ms-font-size-xl;
+      opacity: 1;
```

Comment:
Was all this intentional? Seems like a pretty dramatic change.

Reply:
yes this was intentional, the previous styling would make the svg icons come out very small, and not be visible at all, since they are wrapped in a `span` element

Reply:
I realized later that this styling was originally intended for showing an icon label on hover, and this change entirely removes that code. So I think we need to bring that back but with more specific selectors for showing/hiding.